### ISSUE TITLE
[Bug] Recipe lookup doesn't work

### ISSUE THREAD

**Issue by memphismc** [shedaniel/RoughlyEnoughItems#1500]
### What happened?

when trying to search in the recipe book nothing pops up, already stated this issue on discord but i opened it here for easier tracking

### What mod loaders are you seeing the problem on?

Fabric

### What do you think this bug is of?

- [ ] Visual
- [X] Recipe Lookup
- [ ] Cheat Mode
- [ ] Plugin Integration / JEI Plugin Compatibility
- [ ] Others

### Relevant log output

```shell
rei.log: https://pastebin.com/vANM0SEQ

latest.log: https://pastebin.com/vutC58Fc
```

### Anything else?

my mods folder: https://drive.google.com/file/d/1rJjTzuljtg_XCCPT2C3ZcNoTQJQeXStO/view?usp=sharing

### By submitting this issue, I have included the necessary logs by pasting the contents into the correct location or attaching the file as an upload.

- [X] Yes, and I did not use any paste services other than GitHub Gists.

### By submitting this issue, I have confirmed my REI and REI's dependencies are up to date.

- [X] Yes

**Comment by nightpool** [comment]
I'm seeing the same issue with the vanilla recipe book. It works when I disable REI

**Comment by memphismc** [comment]
> I'm seeing the same issue with the vanilla recipe book. It works when I disable REI

just switch to EMI at this point, been waiting too long for this issue to get fixed and it still hasn't been fixed :/

**Comment by nightpool** [comment]
Hi, it looks like there was a comment censored from this discussion. I think that was inappropriate, it should be possible for users to discuss possible alternatives when a major issue is preventing them from using the mod.

**Comment by shedaniel** [comment]
I agree that was inappropriate, but I will continue to not allow that. You can just interpret that as I am weak or something. I don't think your comment was helpful in anyway in solving the actual issue.

Sadly, I don't have much free time to figure out these issues one-on-one personally, given that this is a special case involving select mod combinations. Please understand that I am only one person, and I don't do modding full time.

The problem here is that REI depends on vanilla books working, and because REI does not change any behaviours regarding vanilla books, REI is not causing the issue. It is most likely some other mods that is causing this issue only when REI is installed. I am not outright saying REI isn't at fault though.
